confirm_big_donations.js

```diff
@@ -6,8 +6,13 @@ $(document).ready(function() {
    */
   $('form').data('confirmed', false).submit(function(event) {
     var $splitTotal = $('#total-to-split');
-    var $other = $('#amount_other_field');
-    var donationAmount = parseFloat($other.val());
+    var donationAmount = 0;
+    var amountChecked = $('input:radio[name=amount]:checked').val();
+    if (amountChecked.length > 0) {
```

Comment:
Oops, that's a bug. Thanks for catching it! It should be (and now is with the latest commit) checking the length of the jQuery selector, i.e. checking whether any button values are selected. The end result is the same, since the length of the value of zero items is also zero. But worth fixing anyway.